actionview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Rename `ActionView::TestCase::Behavior::{Content,RenderedViewContent}`
```

Comment:
@seanpdoyle Could you expand this changelog to make sense in English? I understand what it means, but someone with limited bash-fu might not be looking for this

Reply:
@zzak I've opened https://github.com/rails/rails/pull/50752.